Podfile

```diff
@@ -8,11 +8,11 @@ def shared
   pod 'CocoaLumberjack', '~> 3'
   pod 'Reachability', '~> 3'
   pod 'Sentry', '~> 4'
-  pod 'Sodium-Fork', '~> 0.8.2'
+  pod 'Sodium', '~> 0.9.1'
```

Comment:
This could be `~> 0.9` as well – CocoaPods won't downgrade the version.

Reply:
Agreed to `>= 0.9.1` in Slack.